From ea7120ed901bf6161bb483ab73211a6491daac8f Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Sun, 16 Feb 2020 18:05:06 +0100
Subject: Check activeModule before using it

`activeModule` can be a nullptr here, as this is outside/after the
`if (activeModule)`.
This causes kontact to crash when opening its settings.

BUG: 417396
FIXED-IN: 5.68.0
Differential Revision: https://phabricator.kde.org/D27433
---
 src/kcmultidialog.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kcmultidialog.cpp b/src/kcmultidialog.cpp
index 90da1c0..6a36ba5 100644
--- a/src/kcmultidialog.cpp
+++ b/src/kcmultidialog.cpp
@@ -181,7 +181,7 @@ void KCMultiDialogPrivate::_k_clientChanged()
 #endif
     }
 
-    auto buttons = activeModule->buttons();
+    auto buttons = activeModule ? activeModule->buttons() : KCModule::NoAdditionalButton;
 
     QPushButton *resetButton = q->buttonBox()->button(QDialogButtonBox::Reset);
     if (resetButton) {
-- 
cgit v1.1
